tests/campaigns/test_integration.py

```diff
@@ -444,3 +467,33 @@ def setUp(self):
 
     def test_returns_id(self):
         self.assertIn('id', self.response)
+
+
+class TestRefillSearchCampaignSubscription(BaseSubscriptionTestCase):
+    """
+    SiteWitService.refill_search_campaign_subscription()
+    """
+
+    def setUp(self):
+        self.response = self.service.refill_search_campaign_subscription(
+            self.account_token, self.campaign_id, 510, 500, 'USD',
+            datetime.utcnow().date() + timedelta(32))
+
+    def test_refills_subscription_for_given_amount(self):
+        self.assertEqual(self.response['charge']['items'][1]['price'], 510)
+
+
+class TestRefillDisplayCampaignSubscription(BaseSubscriptionTestCase):
```

Comment:
DRY. We DRYed this in many tests above :)

Reply:
Sure, forgot to do that for that case - will do.